presto-test-jdbc-compatibility/src/test/java/io/prestosql/TestJdbcCompatibility.java

```diff
@@ -283,8 +284,7 @@ public void testSelectMongoObjectId()
         String query = "SELECT ObjectId('55b151633864d6438c61a9ce') AS objectId";
         checkRepresentation(query, ((resultSet, columnIndex) -> {
             assertThat(resultSet.getObject(columnIndex)).isEqualTo(new byte[]{85, -79, 81, 99, 56, 100, -42, 67, -116, 97, -87, -50});
-            assertThat(resultSet.getString(columnIndex)).isEqualTo("");
-            assertThat(resultSet.getMetaData().getColumnType(columnIndex)).isEqualTo("");
+            assertThat(resultSet.getMetaData().getColumnType(columnIndex)).isEqualTo(JAVA_OBJECT);
```

Comment:
nit: JAVA_OBJECT could be a parameter to checkRepresentation and this be tested for all cases